src/main/java/nextstep/fp/Lambda.java

```diff
@@ -27,30 +28,22 @@ public void run() {
     }
 
     public static int sumAll(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            total += number;
-        }
-        return total;
+        return numbers.stream()
+            .mapToInt(e -> e)
+            .sum();
     }
 
     public static int sumAllEven(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            if (number % 2 == 0) {
-                total += number;
-            }
-        }
-        return total;
+        return numbers.stream()
+            .filter(e -> e % 2 == 0)
+            .mapToInt(e -> e)
+            .sum();
     }
 
     public static int sumAllOverThree(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            if (number > 3) {
-                total += number;
-            }
-        }
-        return total;
+        return numbers.stream()
+            .filter(e -> e > 3)
+            .mapToInt(e -> e)
+            .sum();
```

Comment:
요구사항은 람다를 활용해 중복을 제거하는 것인데요
형태만 다를 뿐 중복 코드가 존재하고 있어요
실습에서 요구하는 람다를 활용한 중복 제거가 아니라고 생각해요 

힌트를 꼭 참고하셔서 Conditional 클래스와 sumAll 메서드를 추가해서 아래처럼 중복을 제거해 보시면 좋을 것 같아요

```suggestion
        return sumAll(numbers, number -> number > 3);
```

Reply:
앗 이런식으로 중복제거를 하는거였군요 감사합니다!!